dbt/src/main/kotlin/com/getstrm/pace/dbt/ModelWriter.kt

```diff
@@ -24,7 +29,10 @@ class ModelWriter(private val policy: DataPolicy, private val sourceModel: DbtMo
      */
     @VisibleForTesting
     fun targetFilePath(target: DataPolicy.Target): String {
-        return sourceModel.originalFilePath.substringBeforeLast("/") +
+        return sourceModel.originalFilePath
+            // Windows uses backslashes, so we need to replace them with forward slashes
+            .replace("\\", "/")
+            .substringBeforeLast("/") +
```

Comment:
Does this also work when the jar is executed directly?

Reply:
It should, as this is part of the manifest and therefore does not differ from executing via the Docker image